.prow/ccm-migrations.yaml

```diff
@@ -36,7 +36,7 @@ presubmits:
             - name: PROVIDER
               value: gcp
             - name: VERSION_TO_TEST
-              value: '1.28'
+              value: '1.29'
```

Comment:
This might simply not work anymore, since the in-tree CCMs are gone. If in question, it migh be time to get rid of the ccm-migration-tests alltogether.

Reply:
done!